### ISSUE TITLE
Java / JDownloader2 implementation is broken

### ISSUE THREAD

**Issue by Silejonu** [martinpl/tray-icons-reloaded#20]
Distribution: Arch Linux & Fedora 33
So far, the following tray icons do not show:
Deluge (works in X11, not Wayland)
Battle.net through Wine (works in X11, not Wayland)
JDownloader2

**Comment by martinpl** [comment]
Yes, Wayland is not supported. JDownloader2 is interesting, it is clearly broken for GNOME 3 API, but could be hacked to work state. 

Two things need to be done: 
- Black icons need to replace with something appropriate (left on screenshot)
- Icon window need to be hidden (right on screenshot) (may require external tool)

![image](https://user-images.githubusercontent.com/8329810/106389328-0d7db400-63e3-11eb-92b2-acb79d0b358b.png)

**Comment by dCo3lh0** [comment]
yes this systray icon is broken for me as well, i've not even configured to close to systray because if i do i cant call jdownloader up again and its frozen running on the background.
